### PR TITLE
[General Reorganization] New Folder: Developer Tools

### PR DESCRIPTION
**Pull request by caupolicandiaz**
### Description

This reorganization PR will consolidate some general terms under the classification- Developer Tools.

Entries to be moved include:
IDE, Code-editor and Abstract syntax tree

Additionally, a new issue will be created for a 'version control' entry that will also be part of this group.

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/developer-tools/terms/developer-tools.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Infrastructure'
+CatalogContent:
+  - 'paths/computer-science'
+  - 'paths/code-foundations'
+---
+
+**Developer tools** are the technologies used in the development of software. Although, the suite of applications used by a team or individual developer can vary significantly there are a number of classes of tools that are broadly adopted in the service of creating software applications. Below are common tools leveraged by software developers:
```
Comment (KTom101) [inline content/general/concepts/developer-tools/terms/developer-tools.md:14]:
```suggestion
**Developer tools** are the technologies used in the development of software. Although, the suite of applications used by a team or individual developer can vary significantly there are several classes of tools that are broadly adopted in the service of creating software applications. Below are some common tools leveraged by software developers:
```

**Review by KTom101 — COMMENTED** [review]
@caupolicandiaz I left a small change.

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz I left a small change.

I've incorporated your suggestions. Thanks

File: content/general/concepts/developer-tools/terms/developer-tools.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Infrastructure'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/code-foundations'
+---
+
+**Developer tools** are the technologies used in the development of software. Although, the suite of applications used by a team or individual developer can vary significantly there are several classes of tools that are broadly adopted in the service of creating software applications. Below are some common tools leveraged by software developers:
```
Comment (HishamT) [inline content/general/concepts/developer-tools/terms/developer-tools.md:14]:
```suggestion
**Developer tools** are the technologies used in the development of software. Although the suite of applications used by a team or individual developer can vary significantly, there are several classes of tools that are broadly adopted in the service of creating software applications. Below are some common tools leveraged by software developers:
```

**Review by HishamT — APPROVED** [review]
Looks Good! Left one comment.

**Comment by caupolicandiaz** [comment]
> Looks Good! Left one comment.

Thanks, it's been updated.

**Review by KTom101 — APPROVED** [review]

**Comment by KTom101** [comment]
@caupolicandiaz I think the file path for "developer-tools.md" is wrong. `content/general/concepts/developer-tools/terms/developer-tools.md` should be `content/general/concepts/developer-tools/developer-tools.md`, right?

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz I think the file path for "developer-tools.md" is wrong. `content/general/concepts/developer-tools/terms/developer-tools.md` should be `content/general/concepts/developer-tools/developer-tools.md`, right?

Good catch. I must have accidentally selected the terms folder when I created it.

**Comment by KTom101** [comment]
@caupolicandiaz Is this good to go?

**Pull request merged**